gluoncv/data/mscoco/detection.py

```diff
@@ -104,6 +108,18 @@ def annotation_dir(self):
         """
         return 'annotations'
 
+    def get_im_aspect_ratio(self):
```

Comment:
anyway to get it from coco json? Read raw image is pretty slow

Reply:
PIL only read the image header so it is not that slow, but json could be faster